backend/utils/error.ts

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
If suggestion in https://github.com/Geeks-Academy/structure/pull/4/files#r602778144 will work, we dont need this class anymore